docs/_documentations/troubleshooting.md

```diff
@@ -390,6 +390,33 @@ These steps reproduce the issue:
 
 **Workaround** Run the `Attach Debugger` action manually.
 
+<!--
+Codewind version: 0.6.0
+Action/Topic: Appsody with Codewind
+Issue type: bug/info
+Issue link: https://github.com/eclipse/codewind/issues/1061
+-->
+## Appsody binds fail
+
+If an Appsody respository is already added in your local Appsody CLI, and if you try to add the Appsody repository again with a different name or ID, the Appsody bind might fail in the following scenario:
+1. Use a template manager to add the Appsody repository in Codewind.
+2. Try to bind the project to a stack from the added Appsody repository.
+3. Create the project in Codewind.
+The `appsody.log` displays this error:
+```
+[Error] The current directory is not a valid appsody project. Run `appsody init <stack>` to create one. Run `appsody list` to see the available stacks.
+```
+The `cwctl` log path in the Codewind log in VS Code displays an error like this one:
```

Comment:
The `cwctl` log "path"?  Or is it supposed to be "part"?

Reply:
I can refer to it just as "log" if that's OK.
Also, I see line 409 refers to VS Code--does the error only appear in VS Code, or does this whole issue just apply to VS Code (and not to Eclipse)?

Reply:
Come to think of it, I'm not sure if users know what is `cwctl` since we haven't publicized it for users to use... 

This issue applies to both VS Code and Eclipse.  I almost think we should say that Codewind will "show" this error, and then in brackets add, in VS Code this is shown in the Codewind log

Reply:
Yeah, `cwctl` isn't something the user is aware of.